src/defaults/definitions.js

```diff
@@ -38,5 +38,25 @@ module.exports = {
     period: 60,
     evaluationPeriods: 1,
     comparisonOperator: 'GreaterThanThreshold',
-  }
-};
\ No newline at end of file
+  },
+  bunyanWarnings: {
```

Comment:
These are just defaults that can be specified, but don't have to be. Their individual attributes are over-writable in your serverlesss.yml file. In addition you can add your own specific ones detailing all the fields. I thought if a lot of our services will be using bunyan, it is worth having a default warning and error metric to reduce the work and noise in the serverless.yml file.